### PR TITLE
Search service

### PR DESCRIPTION
**Pull request by Paramveer12**
Screenshot_20200606_145738

### CODE REVIEW

**Comment by wayne9499** [comment]
screenshot(s)?

File: src/app/tasks-layout/tasks-layout.component.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ export class TasksLayoutComponent implements OnInit {
   H2: string;
   H3: string;
   H4: string;
+  isVisible: boolean = false;
+  
 
   @Input() heading: Text;
```
Comment (wayne9499) [inline src/app/tasks-layout/tasks-layout.component.ts:26]: where are you using it??

Comment (Paramveer12) [reply src/app/tasks-layout/tasks-layout.component.ts:26]: to hide the button i guess

**Comment by lakshay-angrish** [comment]
@Paramveer12 Could you add such visibility markers to due today, upcoming and completed divs? It should be false when the number of tasks in their list is zero and true otherwise.

**Comment by Paramveer12** [comment]
> @Paramveer12 Could you add such visibility markers to due today, upcoming and completed divs? It should be false when the number of tasks in their list is zero and true otherwise. Also, hide "All Tasks".

K

**Pull request merged**